app/services/firestore.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  DocumentReference,
```

Comment:
it would be nice to  specify `type` for DocumentReference

Reply:
Good catch! I totally missed that one.